scalameta/tests/jvm/src/test/resources/semanticdb.expect

```diff
@@ -88,6 +84,16 @@ Synthetics:
   [13..16): Int => _root_.scala.reflect.ClassTag.Int.
 
 
+scalameta/semanticdb-integration/src/main/scala/example/Example.scala
```

Comment:
I think we should accommodate this case in `Database.load` (org.langmeta.internal.semanticdb) so that this document with only messages is merged with the one above. Something like this should do the trick
```scala
    private def mergedDatabase = {
      def isOnlyMessages(sdocument: s.Document) =
        sdocument.messages.nonEmpty &&
            sdocument.names.isEmpty &&
            sdocument.synthetics.isEmpty &&
            sdocument.contents.isEmpty &&
            sdocument.symbols.isEmpty
      sdatabase.documents match {
        case Seq(doc, messages) if isOnlyMessages(messages) =>
          doc.copy(messages = doc.messages ++ messages.messages)
        case _ => sdatabase
      }
    }
```

Reply:
This will slow down loading. Does it buy us much?

Reply:
One more design point. If we decide that we want to merge, we could merge all documents that have the same input. That looks like a less arbitrary criterion.